core/src/test/scala/com/microsoft/azure/synapse/ml/core/test/fuzzing/Fuzzing.scala

```diff
@@ -213,9 +213,9 @@ trait ExperimentFuzzing[S <: PipelineStage] extends TestBase with DataFrameEqual
     }
   }
 
-  test("Experiment Fuzzing") {
-    testExperiments()
-  }
+  // test("Experiment Fuzzing") {
```

Comment:
You can just delete these. Thanks for finding this!

Reply:
I've dropped these changes out as they were just a temporary test-speed up for my local dev cycle